stdlib/LinearAlgebra/src/diagonal.jl

```diff
@@ -325,6 +325,11 @@ ldiv!(adjD::Adjoint{<:Any,<:Diagonal{T}}, B::AbstractVecOrMat{T}) where {T} =
 ldiv!(transD::Transpose{<:Any,<:Diagonal{T}}, B::AbstractVecOrMat{T}) where {T} =
     (D = transD.parent; ldiv!(D, B))
 
+function ldiv!(D::Diagonal, A::Union{LowerTriangular,UpperTriangular})
+    broadcast!(/, parent(A), parent(A), diag(D))
```

Comment:
I forgot the reshape here. Needs for be `reshape(D.diag, 1, length(D.diag))` instead. (I recently learned that `diag` now copies.)

Reply:
Happy to add this, but can you please explain why `reshape` is needed here? In any case it should be
```julia
reshape(D.diag, length(D.diag), 1) 
```
for the `ldiv!`, but wouldn't using just the vector `D.diag` be equivalent?

Reply:
I saw you were using `/` and thought this was `rdiv!`. This should then be
```julia
broadcast!(\, parent(A), D.diag, parent(A))
```
`d\a` and `a/d` is not always the same. E.g. if the elements are matrices.